blockchain/blockchain-wallet-v4-frontend
# fix(create bch wallet on btc wallet creation without needing relogging)

## Description
778
Fetch bch metadata on bitcoin wallet creation to create the corresponding bch wallet immediately.
## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
Go to Wallets & Addresses, add a bitcoin wallet and go to bch wallets page. See that the corresponding bch wallet has been created.

## PR Creator Checklist

## PR Reviewer Checklist
